pkg/restore/restore.go

```diff
@@ -653,6 +654,40 @@ func (ctx *context) shouldRestore(name string, pvClient client.Dynamic) (bool, e
 	return shouldRestore, err
 }
 
+// crdAvailable waits for a CRD to be available for use before letting the restore continue.
+func (ctx *context) crdAvailable(name string, crdClient client.Dynamic) (bool, error) {
+	crdLogger := ctx.log.WithField("crdName", name)
+
+	var available bool
+	err := wait.PollImmediate(time.Second, ctx.resourceTerminatingTimeout, func() (bool, error) {
```

Comment:
`resourceTerminatingTimeout` is 10min by default - wondering if maybe we want to use a shorter timeout here? I'm not sure without looking at some examples.

Reply:
Yeah, that time out is the same one we use for PVs, and it's probably too long for CRs. I wanted to use the same one so that we could allow users to customize without providing too many knobs.